app/components/ContextMenu/Template.js

```diff
@@ -130,7 +130,7 @@ function Template({ items, ...menu }: Props): React.Node {
           disabled={item.disabled}
           selected={item.selected}
           level={item.level}
-          target="_blank"
```

Comment:
Oh yes, I meant to mention this – I don't think this is the right coupling though, It could be possible to use the `level` prop and want an external link. The correct coupling would be to check if `item.href.startsWith("#")`

Reply:
Nice catch! I should have given more thought.